synapseclient/client.py

```diff
@@ -2172,6 +2172,113 @@ def getTeamMembers(self, team):
+        to the invitee.
+
+        :param teamId: Synapse teamId
+        :param inviteeId: Synapse username or profile id of user
+        :param inviteeEmail: Email of user
+        :param message: Additional message for the user getting invited to the
+                        team. Default to None.
+
+        :returns: MembershipInvitation
+        """
+
+        invite_request = {'teamId': str(teamId),
+                          'message': message}
+        if inviteeEmail is not None:
+            invite_request['inviteeEmail'] = str(inviteeEmail)
+        if inviteeId is not None:
+            invite_request['inviteeId'] = str(inviteeId)
+
+        response = self.restPOST("/membershipInvitation",
+                                 body=json.dumps(invite_request))
+        return response
+
+    def invite_to_team(self, team, force=False, **kwargs):
+        """Invite user to a Synapse team via Synapse username or email
+        (choose one or the other)
+
+        :param syn: Synapse object
+        :param team: A :py:class:`synapseclient.team.Team` object or a
+                     team's ID.
+        :param force: If an open invitation exists for the invitee,
+                      the old invite will be cancelled. Default to False.
+        :param inviteeId: Synapse username or profile id of user
+        :param inviteeEmail: Email of user
+        :param message: Additional message for the user getting invited to the
+                        team. Default to None.
+
+
+        :returns: MembershipInvitation or None if user is already a member
+        """
+        user = kwargs.get("inviteeId")
+        email = kwargs.get("inviteeEmail")
+        if email is not None and user is not None:
+            raise ValueError("Must only specify 'user' or 'email'")
+
+        teamid = id_of(team)
+        is_member = False
+        open_invitations = self.get_team_open_invitations(teamid)
+
+        if user is not None:
+            userid = self.getUserProfile(user)['ownerId']
+            membership_status = self.get_membership_status(userid, teamid)
+            is_member = membership_status['isMember']
+            open_invites = [invitation for invitation in open_invitations
+                            if invitation.get('inviteeId') == user]
+        else:
+            open_invites = [invitation for invitation in open_invitations
+                            if invitation.get('inviteeEmail') == email]
+        # Only invite if the invitee is not a member and
+        # if invitee doesn't have an open invitation unless force=True
+        if not is_member and (not open_invites or force):
+            # Delete all old invitations
+            for invite in open_invites:
+                self._delete_membership_invitation(invite['id'])
+            return self.membership_invitation(teamid, **kwargs)
+        # Return None if no invite is sent.  Should an error should be thrown?
```

Comment:
remove the `Should an error should be thrown?` comment. Returning none makes sense

```suggestion
        # Return None if no invite is sent.
```